.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: elixir
+elixir:
+  - 1.7
+otp_release:
+  - 21.0
+sudo: false
```

Comment:
I forgot about it when you added `.travis.yml` to caps, but the default script is not sufficient, since format checker should be run as well, please add this:
```
script:
  - mix format --check-formatted
  - mix test
```
You can do this also in caps